install_heroku_script.sh

```diff
@@ -0,0 +1,9 @@
+cd iframely/
+npm install
+echo "---------------1-------------------"
+cd ../
+cd oauth-proxy/
+npm install
+echo "------------------------------2--------------------------"
+cd ../
+gulp prod
```

Comment:
`gulp dev` contains unminified javascript files, good for development and debugging takes time to load
`gulp prod` creates the files for being hosted on a web server
`gulp live` runs the web server and the application in an environment between dev and prod, minifies files